src/components/AddObsModal.js

```diff
@@ -13,8 +13,8 @@ import { useTranslation } from "sharedHooks";
 import useStore from "stores/useStore";
 
 type Props = {
-  closeModal: ( ) => void,
-  navAndCloseModal: ( string, ?{ camera: string } ) => void
+  closeModal: any,
+  navAndCloseModal: any
```

Comment:
This seems like a real loss of specificity. `( ) => void` doesn't trigger any alerts as a type for me, and IMO,

```
// eslint-disable-next-line @typescript-eslint/ban-types
navAndCloseModal: Function
```

at least signals to devs that this prop is supposed to be a function and not a boolean or some non-function primitive.